src/app/scripts/cac/pages/cac-pages-home.js

```diff
@@ -267,6 +269,10 @@ CAC.Pages.Home = (function ($, ModeOptions,  MapControl, TripOptions, SearchPara
                 tabControl.setTab(tabControl.TABS.DIRECTIONS);
             }
         }
+
+        if (key === 'origin') {
```

Comment:
This would make more sense in the `onTypeheadSelected` of the controller that contains `getNearbyPlaces`. Both Directions and Explore already have `onTypeaheadSelected` methods.